Common/ExecutionModel/vtkThreadedImageAlgorithm.cxx

```diff
@@ -25,12 +25,105 @@
 #include "vtkPointData.h"
 #include "vtkStreamingDemandDrivenPipeline.h"
 
+#include "vtkExtentTranslator.h"
+
+
+
+//---- option 2
```

Comment:
I recommend using an ifdef or even just "#if 0" and "#if 1" for temporary optional code like this.